docs/tutorials/classification.ipynb

```diff
@@ -2,195 +2,433 @@
+      ],
+      "text/plain": [
+       "+-----+----------------------+--------+----------+---+----------+-------+---------------+----------+\n",
+       "|  id | name                 | sex    |      age | … |     fare | cabin | port_embarked | survived |\n",
+       "| --- | ---                  | ---    |      --- |   |      --- | ---   |           --- |      --- |\n",
+       "| i64 | str                  | str    |      f64 |   |      f64 | str   |           str |      i64 |\n",
+       "+==================================================================================================+\n",
+       "|   0 | Abbing, Mr. Anthony  | male   | 42.00000 | … |  7.55000 | null  |   Southampton |        0 |\n",
+       "|   1 | Abbott, Master.      | male   | 13.00000 | … | 20.25000 | null  |   Southampton |        0 |\n",
+       "|     | Eugene Joseph        |        |          |   |          |       |               |          |\n",
+       "|   2 | Abbott, Mr. Rossmore | male   | 16.00000 | … | 20.25000 | null  |   Southampton |        0 |\n",
+       "|     | Edward               |        |          |   |          |       |               |          |\n",
+       "|   3 | Abbott, Mrs. Stanton | female | 35.00000 | … | 20.25000 | null  |   Southampton |        1 |\n",
+       "|     | (Rosa Hun…           |        |          |   |          |       |               |          |\n",
+       "|   4 | Abelseth, Miss.      | female | 16.00000 | … |  7.65000 | null  |   Southampton |        1 |\n",
+       "|     | Karen Marie          |        |          |   |          |       |               |          |\n",
+       "+-----+----------------------+--------+----------+---+----------+-------+---------------+----------+"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "from safeds.data.tabular.containers import Table\n",
     "\n",
-    "titanic = Table.from_csv_file(\"data/titanic.csv\")\n",
+    "raw_data = Table.from_csv_file(\"data/titanic.csv\")\n",
     "#For visualisation purposes we only print out the first 15 rows.\n",
-    "titanic.slice_rows(0, 15)"
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "execution_count": null,
-   "outputs": []
+    "raw_data.slice_rows(0, 5)"
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "2. Split the titanic dataset into two tables. A training set, that we will use later to implement a training model to predict the survival of passengers, containing 60% of the data, and a testing set containing the rest of the data.\n",
-    "Delete the column `survived` from the test set, to be able to predict it later:"
+    "### Removing olumns with high idness / stability / missing value ratio"
```

Comment:
```suggestion
    "### Removing Low-Quality Columns
```

Your headings should be more abstract and concise. Currently, the table of content is quite bloated:

![image](https://github.com/Safe-DS/Library/assets/2501322/019b06ed-083f-4991-80d4-3537bae82048)